src/index.js

```diff
@@ -0,0 +1,136 @@
+module.exports = {
+  root: true,
+  rules: {
+    quotes: 0,
```

Comment:
Je trouve ça assez inconsistant le manque des guillemets (je sais qu'ils sont optionnels pour ce genre de cas, mais je trouve ça néanmoins assez frustrant)
```suggestion
    "quotes": 0,
```